treeherder/log_parser/crossreference.py

```diff
@@ -114,7 +114,12 @@ def __call__(self, failure_line):
         else:
             action = failure_line.action
 
-        msg = getattr(self._formatter, action)(as_dict(failure_line)).split("\n", 1)[0]
+        try:
+            f = getattr(self._formatter, action)
+        except AttributeError:
+            return None
+
+        msg = f(as_dict(failure_line)).split("\n", 1)[0]
```

Comment:
Thank you for fixing this :-)

Whilst it's now slightly more readable, I do still find it a bit too much on the terse/painful to read side (it sometimes feels like there's a competition going to see how many functions we can nest on one line ;-)) - but fine with leaving as is for now.